.shippable.yml

```diff
@@ -0,0 +1,18 @@
+build_image: freedomjs/freedom
+language: node_js
+node_js:
+- '0.10'
+notifications:
+  email:
+    recipients:
+      - freedom@cs.washington.edu
+    on_success: change
+    on_failure: always
+before_install:
+  - "export DISPLAY=:10.0"
+  - "Xvfb :10 -screen 0 1280x1024x24 &"
+install:
+  - "git pull"
```

Comment:
Unfortunately I don't think so. We've specified the build_image to be freedomjs/freedom (This uses master as the default branch). However, both Docker Hub and Shippable are configured to trigger a build when the push happens. There is no guarantee that the Docker image that is pulled by Shippable will include the commits from the latest push. I believe my current setup only works on builds from master and it's a bit undetermined at the moment what happens when builds are triggers from branches or pull requests.